CppCoreGuidelines.md

```diff
@@ -10254,7 +10254,7 @@ Such names are commonly used for macros. Thus, `ALL_CAPS` name are vulnerable to
     // somewhere else in some other header:
     enum Coord { N, NE, NW, S, SE, SW, E, W };
 
-    // somewhere third in some poor programmer's .cpp:
+    // somewhere third in some bad programmer's .cpp:
```

Comment:
used "unfortunate"

Reply:
I'm not a native english speaker, and I think "unfortunate" is a fine change, but not as good as "poor". The word "poor" is overloaded. This instance invokes the third definition (from [merriam webster](https://www.merriam-webster.com/dictionary/poor)); "exciting pity". I'm not aware of any other english words with the same meaning (and if there is one, I would expect other non-native english speakers not be familiar with it).

I think all other places where you change "poor" to "bad" are clear improvements though.

Reply:
To a casual reader it may also appear to be the definition "Of low quality" implying that this section of the code was bad due to the low quality of its author. It seems that's what happened with the original change to "bad". I think it's more important to clarify that the fault is not in this chunk of code than it is to consider the impact that has on the author of that code. By that heuritstic I think "unfortunate" is preferable to "poor".